### ISSUE TITLE
Is the data publicly available somewhere?

### ISSUE THREAD

**Issue by apoorvpatne10** [JosePaniagua7/vowels-lip-reading#1]
I'm seeing the training dataset provided is in the form of json. Did you create it or is it publicly available in the form of frames/videos? 

Hope you guys know english.

**Comment by JosePaniagua7** [comment]
Hello @apoorvpatne10  we are glad you found our repo! 

Because of the data treatment law we cannot upload the pics that we have used for building the training set, but, here are the steps to build it:
 
1) Took pics of 15 different individuals vocalizing the vowels in order, taking 20 frames per second
2) We store those pics in a tree shape folder structure root > a >pics   , root>e>pics
3) After that, we iterate through them passing running the file "DataTreatment/get_images_key_points.py"
4) With the X,Y generated key points  we decided to calculate the distance between the pixels 
5) We use the calculated distance as input in out neuronal network. 

We are looking forward to hear news about your usage of our project! 
Regards!

**Comment by apoorvpatne10** [comment]
> Took pics of 15 different individuals vocalizing the vowels in order, taking 20 frames per second

So what you mean is we have to record a ~1 sec video of a person saying some vowel (say 'a'). This video will have 20 frames. So 20 pics which represents the vowel ('a') said by the person. And we shall repeat the same process for multiple individuals. 

Hope I'm right?

**Comment by JosePaniagua7** [comment]
@apoorvpatne10 That is right, although, I'll recommend you to use the function "burst" of your mobile phone camera, so you don't have to process videos but images instead. 

We are looking forward to hear news about your usage of our project!
Regards!
